<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginsControlGUIClass</class>
 <widget class="QMainWindow" name="PluginsControlGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plugins...</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Available plugins</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="lstPlugins">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectItems</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="resizeMode">
           <enum>QListView::Adjust</enum>
          </property>
          <property name="viewMode">
           <enum>QListView::ListMode</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>225</width>
           <height>185</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Name :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="lblName">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Author :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="lblAuthor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="font">
               <font>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Contact :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="lblContact">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>Description :</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="lblDescription">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="5">
         <widget class="QPushButton" name="btnConfigure">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Configure...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="btnReset">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="chkEnabled">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Enable</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnConfigure</sender>
   <signal>clicked()</signal>
   <receiver>PluginsControlGUIClass</receiver>
   <slot>configurePlugin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkEnabled</sender>
   <signal>clicked()</signal>
   <receiver>PluginsControlGUIClass</receiver>
   <slot>enableDisablePlugin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReset</sender>
   <signal>clicked()</signal>
   <receiver>PluginsControlGUIClass</receiver>
   <slot>resetPlugin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstPlugins</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>PluginsControlGUIClass</receiver>
   <slot>pluginSelected(QListWidgetItem*,QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pluginSelected(QListWidgetItem*,QListWidgetItem*)</slot>
  <slot>configurePlugin()</slot>
  <slot>enableDisablePlugin()</slot>
  <slot>resetPlugin()</slot>
 </slots>
</ui>
